src/main/java/bio/terra/service/rawls/RawlsClient.java

```diff
@@ -27,10 +26,9 @@ public class RawlsClient {
   private final HttpHeaders headers;
 
   @Autowired
-  public RawlsClient(RawlsConfiguration rawlsConfiguration) {
+  public RawlsClient(RawlsConfiguration rawlsConfiguration, RestTemplate restTemplate) {
     this.rawlsConfiguration = rawlsConfiguration;
-    this.restTemplate = new RestTemplate();
-    restTemplate.setRequestFactory(new HttpComponentsClientHttpRequestFactory());
```

Comment:
Flagging for reviewers: is there a reason that we weren't using the common `RestTemplate` bean before?

Reply:
Nope, I think that got copy pasted from the Bard client

Reply:
🍝